charts/velero/templates/backupstoragelocation.yaml

```diff
@@ -1,4 +1,43 @@
 {{- if .Values.backupsEnabled }}
+{{- if and .Values.configuration.backupStorageLocation (kindIs "slice" .Values.configuration.backupStorageLocation) }}
+{{- range .Values.configuration.backupStorageLocation }}
+---
+apiVersion: velero.io/v1
+kind: BackupStorageLocation
+metadata:
+  name: {{ .name }}
+  namespace: {{ $.Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade,post-rollback
+    "helm.sh/hook-delete-policy": before-hook-creation
+  labels:
+    app.kubernetes.io/name: {{ include "velero.name" $ }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    helm.sh/chart: {{ include "velero.chart" $ }}
+spec:
+  credential: 
+    name: {{ .credential.name }}
+    key: {{ .credential.key }}
```

Comment:
Could you update the values.yaml to indicate how to configure the credential?

Reply:
Let me know if the additional comments are intuitive! I tried to be concise.